examples/api-basic/main.go

```diff
@@ -17,20 +17,35 @@ limitations under the License.
 package main
 
 import (
+	"flag"
+	stdlog "log"
 	"net/http"
 
+	"github.com/caicloud/nirvana"
+
 	"github.com/caicloud/nirvana/examples/api-basic/api/v1"
 	"github.com/caicloud/nirvana/examples/api-basic/api/v2"
+	"github.com/caicloud/nirvana/log"
 	"github.com/caicloud/nirvana/web"
 )
 
 func main() {
+	flag.Parse()
+
 	if err := web.RegisterDefaultEnvironment(); err != nil {
 		panic(err)
 	}
 
-	s := web.NewDefaultServer()
+	s, err := nirvana.New(&nirvana.Config{
```

Comment:
maybe should be `server.New`?

Reply:
Do you mean a new package `github.com/caicloud/nirvana/server` ?